rmgpy/molecule/translator.py

```diff
@@ -509,6 +509,10 @@ def _write(mol, identifier_type, backend):
 
     Returns a string identifier of the requested type.
     """
+    # Check that the molecule is not empty
+    if not mol.atoms:
+        return ''
```

Comment:
My thought is that an empty molecule object is usually a really bad thing, so shouldn't we have a more easily recognizable string than ''?

Reply:
Do you have examples of when it would really bad? Currently, the reason I added this is for when you instantiate an empty Molecule instance and try to do something with it.
```
mol = Molecule()
mol.toSMILES()  # ValueError
mol.toInChI() # ValueError
mol.__repr__() # ValueError
mol.__str__() # ValueError
```
I feel like these should be able to succeed with an empty Molecule.

Reply:
Ok, we talked about this.